anvil-rstudio-bioconductor-devel/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.15.0 - 11/2/2021
+
+- Rebuild stack with latest devel image of Bioconductor with version 3.15
```

Comment:
isn't the latest version of bioc 3.14?

Reply:
Yes, i'm updating two images here. 

latest development version is 3.15 

latest release version is 3.14

Reply:
i don't see any changes to the development dockerfile in this PR - what does rebuilding the image as is update?